index.js

```diff
@@ -113,32 +123,45 @@ function text (scanner) {
     }
     node.value += scanner.next()
   }
+  node.position = { start, end: scanner.position() }
   return node
 }
 
 /*
  * <summary-sep>  ::= "!"? ":" *<whitespace>
  */
 function summarySep (scanner) {
+  const start = scanner.position()
   const node = {
     type: 'summary-sep',
     children: []
   }
   if (isSummarySep(scanner.peek())) {
```

Comment:
If we do keep the `summary-sep` node, perhaps I should have called this `isBreakingSep`?